app/config/views.yml

```diff
@@ -47,13 +48,30 @@ ezpublish:
                         template: full\article.html.twig
                         match:
                             Identifier\ContentType: article
-
+                    bundle_list:
+                        controller: app.controller.bundle:showBundlesListAction
+                        params:
+                            query:
+                                query_type: AppBundle:Children
+                                parameters:
+                                    parent_location_id: '%bundles.location_id%'
+                                    limit: '%bundles.cards_limit%'
+                                    sorting_content_type: bundle
+                                    sorting_field: name
```

Comment:
I don't know if you remember, but we discussed the sort order and we decided to use the `DateModified` of the content item.